versioning/BRM/sessionmanagerserver.h

```diff
@@ -123,10 +123,7 @@ class SessionManagerServer
      * It does not destroy the semaphores.  Those persist until the system
      * is shut down.
      */
-    virtual ~SessionManagerServer()
-    {
-        if (txnidfd >= 0 ) close(txnidfd);
-    }
+    virtual ~SessionManagerServer();
```

Comment:
We probably don't need virtual dtor at all.

Reply:
Now that it does nothing that's true.  We can do a second round of cleanup at some point.